ossindex/ossindex_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2018 Sonatype Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Definitions and functions for processing the OSS Index Feed
+package ossindex
+
+import (
+	"github.com/sonatype-nexus-community/nancy/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
The golang community is kinda torn on whether to use asserts or not (https://golang.org/doc/faq#assertions). You don't have to change anything, as like they even concede they are super useful. Figured I'd share so you can be in on the drama as well, if you weren't :)

Reply:
> The golang community is kinda torn on whether to use asserts or not (https://golang.org/doc/faq#assertions). You don't have to change anything, as like they even concede they are super useful. Figured I'd share so you can be in on the drama as well, if you weren't :)

Interesting. I was not aware of the drama. ;) I guess I equate "assertions" to things like Java [assert](https://docs.oracle.com/javase/8/docs/technotes/guides/language/assert.html) (e.g. something that ships in productions code) rather than unit test "[assert](http://junit.sourceforge.net/javadoc/org/junit/Assert.html).xxx" (something that exists only in test code).

Reply:
Yeah I think for tests it's totally fine, just interesting they didn't build it into code itself with an `assert` keyword or something akin.